docs/deployer/helm.md

```diff
@@ -43,6 +60,22 @@ spec:
 
     updateStrategy: update | patch # optional; defaults to update
 
+    # Configuration of the health checks for the resources.
+    # optional
+    healthChecks:
+      # Allows to disable the default health checks.
+      # optional; set to false by default.
+      disableDefault: true
+      # Defines the time to wait before giving up on a resource
+      # to be healthy. Should be changed with long startup time pods.
+      # optional; default to 60 seconds.
+      timeout: 30s
+
+    # Defines the time to wait before giving up on a resource to be deleted,
+    # for instance when deleting resources that are not anymore managed from this DeployItem.
+    # optional; default to 60 seconds.
+    deleteTimeout: 2m
```

Comment:
I would also increased that default timeout

Reply:
See above.